src/main/features/response/views/evidence.njk

```diff
@@ -1,5 +1,9 @@
 {% extends "layout.njk" %}
-{% from "form.njk" import csrfProtection, linkButton, textArea, saveAndContinueButton, errorSummary %}
+{% from "form.njk" import csrfProtection, linkButton, saveAndContinueButton, errorSummary %}
+
+{% block additional_scripts %}
+  <script src="{{ asset_paths['js'] }}/select-toggled-content.js"></script>
```

Comment:
The way script is written makes it usable only for this particular use scenario. Name should reflect it so maybe we could have `envidence-select-handler.js` or similar.

Reply:
renamed